File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/inventory/api/InventoryAPI.java
package com.gmail.borlandlp.minigamesdtools.gui.inventory.api;

import com.gmail.borlandlp.minigamesdtools.gui.inventory.DrawableInventory;
import org.bukkit.entity.Player;

public interface InventoryAPI {
    void show(DrawableInventory drawableInventory, Player player);
    DrawableInventory getInventory(Player player);
    boolean isView(Player player);
    void close(Player player);
    void unregister(Player player);
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/inventory/api/InventoryAPI.kt
package com.gmail.borlandlp.minigamesdtools.gui.inventory.api

import com.gmail.borlandlp.minigamesdtools.gui.inventory.DrawableInventory
import org.bukkit.entity.Player

interface InventoryAPI {
    fun show(drawableInventory: DrawableInventory, player: Player)
    fun getInventory(player: Player): DrawableInventory
    fun isView(player: Player): Boolean
    fun close(player: Player)
    fun unregister(player: Player)
}